platform/ui/src/components/Select/Select.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import ReactSelect from 'react-select';
+
+import './Select.css';
+
+const Select = ({
+  autoFocus,
+  className,
+  classNamePrefix,
+  isDisabled,
+  isMulti,
+  isSearchable,
+  name,
+  onChange,
+  options,
+  placeholder,
+  noOptionsMessage,
+  value,
+}) => {
+  return (
+    <ReactSelect
+      autoFocus={autoFocus}
+      className={classnames(
+        className,
+        'flex flex-col flex-1 hover:cursor-pointer mt-2 customSelect__wrapper'
+      )}
+      classNamePrefix="customSelect"
+      isDisabled={isDisabled}
```

Comment:
It'd be good to add a custom style like `cursor-not-allowed` when `isDisabled` is true.

Reply:
Weird way to update it, but its done :)